### PR TITLE
clean up extraneous old code from sentiment

### PR DESCRIPTION
**Pull request by ntarn**
Delete router.function and export and use getSentimentScore from the sentiment router directly (google cloud natural language api)

### CODE REVIEW

File: search-sentiments-map/routes/search.js [hunk 1]
```diff
@@ -186,16 +187,16 @@ async function updateSearchResults() {
   let countries = countriesJson.map(country => country.id);
 
   const trends = await search.retrieveGlobalTrends();
-  for (let i = 0; i < trends.length; i++) {
+  for (let i = 0; i < trends.length; i++) {
```
Comment (carmenBen) [inline search-sentiments-map/routes/search.js:190]:
```suggestion
  for (let i = 0; i < trends.length; i++) {
```

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

File: search-sentiments-map/routes/search.js [hunk 2]
```diff
@@ -186,16 +187,16 @@ async function updateSearchResults() {
...
     console.log('Creating WorldDataByTopic entity for', topic)
     let countriesData = await search.getSearchResultsForCountriesForTopic(
         countries, topic);
     search.addWorldDataByTopicToDatastore(topic, countriesData);
 
-    // Note: when testing ,use i < 1 to test for only one trend, and comment 
+    // Note: When testing, use i < 1 to test for only one trend, and comment 
     // out `await new Promise` line to avoid 1 minute pauses.
-    await search.sleep(PAUSE_ONE_MIN_MS);
+    // await search.sleep(PAUSE_ONE_MIN_MS);
```
Comment (carmenBen) [inline search-sentiments-map/routes/search.js:199]:
```suggestion
    await search.sleep(PAUSE_ONE_MIN_MS);
```
Unsure why this was commented out.

File: search-sentiments-map/routes/search.js [hunk 3]
```diff
@@ -226,7 +227,7 @@ async function getSearchResultsForCountriesForTopic(countries, topic) {
   await searchInterestsModule.getGlobalSearchInterests(topic)
       .then(async (searchInterests) => {
     // Note: Use i < 3 countries when testing.
-    for (let i = 0; i < countries.length; i++) {
+    for (let i = 0; i < countries.length; i++) {
```
Comment (carmenBen) [inline search-sentiments-map/routes/search.js:230]:
```suggestion
    for (let i = 0; i < countries.length; i++) {
```

File: search-sentiments-map/routes/search.js [hunk 4]
```diff
@@ -307,36 +308,19 @@ async function formatCountryResults(searchResultsJson) {
  * @return {Object} Formatted search result data in JSON form.
  */
 function formatSearchResult(searchResult) {
-  return search.getSentiment(searchResult)
-      .then(response => response.json())
+  return sentiment.getSentimentScore(searchResult.title + searchResult.snippet)
       .then((result) => {
+        console.log('ntarn debug score:' + result);
```
Comment (carmenBen) [inline search-sentiments-map/routes/search.js:313]: Can we delete this print or can we add a TODO comment to delete it later.

File: search-sentiments-map/routes/search.js [hunk 5]
```diff
@@ -307,36 +308,19 @@ async function formatCountryResults(searchResultsJson) {
...
         return {
           title: searchResult.title,
           snippet: searchResult.snippet,
           htmlTitle: searchResult.htmlTitle,
           link: searchResult.link,
-          score: SCORE_SCALE_MULTIPLIER * result.score,
+          score: SCORE_SCALE_MULTIPLIER * result, //result.score
```
Comment (carmenBen) [inline search-sentiments-map/routes/search.js:319]:
```suggestion
          score: SCORE_SCALE_MULTIPLIER * result,
```

File: search-sentiments-map/routes/search.js [hunk 6]
```diff
@@ -307,36 +308,19 @@ async function formatCountryResults(searchResultsJson) {
...
         };
       });
 }
 
-/**
```
Comment (carmenBen) [inline search-sentiments-map/routes/search.js:323]: If you are deleting this, then deleting it on line 409.

File: search-sentiments-map/routes/sentiment.js [hunk 7]
```diff
@@ -52,7 +36,7 @@ async function getSentimentScore(searchResultTitleSnippet) {
     // Detect the sentiment of the text.
     const [result] = await client.analyzeSentiment({ document: document });
     const sentiment = result.documentSentiment;
-
+    console.log('ntarn debug score in sentiment:' + sentiment.score);
```
Comment (carmenBen) [inline search-sentiments-map/routes/sentiment.js:39]: Can we delete this print or can we add a TODO comment to delete it later.

File: search-sentiments-map/routes/sentiment.js [hunk 8]
```diff
@@ -62,4 +46,5 @@ async function getSentimentScore(searchResultTitleSnippet) {
   }
 }
 
-module.exports.router = router;
\ No newline at end of file
+module.exports.router = router;
```
Comment (carmenBen) [inline search-sentiments-map/routes/sentiment.js:49]: If we no longer use router here, we can delete this line and delete line 17 too.

Comment (carmenBen) [reply search-sentiments-map/routes/sentiment.js:49]: Similarly, delete sentiment router from the app.js file.

**Comment by ChenyuHeidiZhang** [comment]
This update creates an issue when ran on cloudshell. My local editor has not been working well lately, so can we wait on merging this until when either the problem is resolved or we don't need cloudshell anymore?

**Pull request merged**